DA-17: Refactor frontend

JeannedArk opened the pull request:
Replace var through const for compiler optimizations and clearity,
user everywhere tableProjects instead of using tableProjects and
rootScope.projects, move sources and add folders for structure,
use locally included libs, general code refactoring.

NicklasLinz commented:
I get the following error:
![da-17](https://cloud.githubusercontent.com/assets/13366276/15148074/f23c1572-16c3-11e6-97e6-c5978cd022fc.png)

JeannedArk committed:
DA-17: Refactor frontend

Replace var through const for compiler optimizations and clearity,
use tableProjects instead of using tableProjects and rootScope.projects,
move sources and add folders for structure, use locally included libs,
general code refactoring.
